components/sidebar/index.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import './index.scss'
+
+const Sidebar = React.memo(({ className }) => {
+  return (
+    <nav className={className}>
+      <ul>
+        <li>
+          <h3>Overview</h3>
```

Comment:
It's not semantically correct to use headings here. Document outline might look weird because of that.